osu.Server.Queues.ScoreStatisticsProcessor/Processors/HitStatisticsProcessor.cs

```diff
@@ -45,15 +45,18 @@ private static void adjustStatisticsFromScore(SoloScore score, UserStats userSta
                 switch (result)
                 {
                     case HitResult.Miss:
+                    case HitResult.LargeTickMiss when score.ruleset_id == 2:
```

Comment:
I'll overlook the usage of `when` here because it's quite legible in such a simple scenario (and would be annoying to write any other way)..

Reply:
Yeah I did that fully well knowing you don't like this syntax purely due to the annoyance factor.